### ISSUE TITLE
High-level documentation priorities

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#760]
We should add more high-level documentation (in general, but also wrt. CTA use cases).

Concrete high-priority suggestions:

- [ ] The basic image analysis example notebook should be extended showing how to do a morpholgy fit [here](http://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/notebooks/image_analysis.ipynb#Morphology-fit)
- [ ] For IACT DL3 data access, we have [data_iact.ipynb](http://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/notebooks/data_iact.ipynb). At the start there should be a section how to do observation selection (e.g. cone search on Crab nebula). For this we should simulate a CTA reference test dataset (given that HESS data release and CTA data challenge are not available).
- [ ] For survey / source detection we have [detect_ts.ipynb](http://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/notebooks/detect_ts.ipynb). This should be extended to produce a catalog as already noted [here](http://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/notebooks/detect_ts.ipynb#Source-catalog)
- [ ] Add a second survey / source detection notebook, that simulates a Galactic plane survey with CTA IRFs (using sky model from gamma-cat), and again produces maps and a simple source catalog.

This is a long-term overview issue to discuss priorities and track status.
Everyone - please give feedback by suggesting other things, help implement that functionality and write those docs.

cc @bkhelifi @cboisson

**Comment by joleroi** [comment]
>  For this we should simulate a CTA reference test dataset (given that HESS data release and CTA data challenge are not available).

IMO the HESS data release should be prioritized over simulating a CTA reference test dataset

**Comment by cdeil** [comment]
@robertazanin and I discussed this a bit just now. With the Gammapy workshops and the CTA data challenge happening in a few weeks, priorities have changed a bit. Basically it's important to extend and improve the documentation now, and to add examples how to do stuff ASAP. There's people from CTA asking if Gammapy can do e.g. a user-defined spectral model, a 2D image based analysis etc. ... stuff we basically already have, but not well explained or documented at all with an example.

Basically I think everyone is already working in that direction. @adonath will add the Fermi reference dataset this week.  Since the HESS data release will come April at the soonest, we have to use HESS or CTA IRFs now (this week, next latest) to generate a better example IACT dataset for the docs.

Let's talk at the end of the week and make a concrete plan what's possible in the given short timescale.

**Comment by cdeil** [comment]
Moving milestone to 0.7 (I'll make the 0.6 release tomorrow)

**Comment by cdeil** [comment]
We have improved our tutorial notebooks, and continue to do so with high priority before the v0.8 release, see #1555 .

We had many of such "improve everything" issues from the past years; in the end I don't think any of them were really helpful / triggered work. So I'm closing this old issue.
